packages/spor-react/src/button/FloatingActionButton.tsx

```diff
@@ -63,12 +63,10 @@ export const FloatingActionButton = ({
       aria-label={children}
       {...props}
     >
-      <Box __css={style.icon}>
-        {icon}
-      </Box>
+      <Box __css={style.icon}>{icon}</Box>
       <MotionBox
         animate={isTextVisible ? "show" : "hide"}
-        initial="show"
+        initial={externalIsTextVisible ? "show" : "hide"}
```

Comment:
This is really the main "fix" here – if we haven't set the `isTextVisible` prop to true, hide the text initially.